### ISSUE TITLE
AS Arena Bug/Abuse

### ISSUE THREAD

**Issue by snowdear** [elementum-games/shinobi-chronicles#385]
Hi,

There is a way to abuse arena if you don`t level up. I believe that this is only relevant for academy students as later on exp requirements are higher.

If a player does not click on profile button it leads to player not lvling up. User can stay lvl1 and just keep beating lvl1 arena opponent. With a little training user can soon 1hit enemy and grind faster to genin, roll BL, if unlucky, start again. I guess this makes road to genin faster than intended.

**Comment by HitoriWoB** [comment]
Level holding is an intended feature. You should be able to be 5 levels below your max level. This can be done by not visiting your profile or toggling the level hold feature in user settings.

**Comment by snowdear** [comment]
Yes, but you can be level 1, while you have enough experience for level 10. Doesn`t this mean that you are 10 levels below max level?

**Comment by HitoriWoB** [comment]
This is still an (small) issue. It's possible to hold levels at a greater value than intended.
Proposed solution:
- Level up on any page
  -  Convert level up messages from raw html header to notification
